src/queryBuilder/Group.js

```diff
@@ -15,32 +15,32 @@ let { background } = styles
 let GroupItem = FilterDragSource(args => {
   let {
     child,
-    tree,
+    node,
     index,
     state,
     root,
     isRoot,
-    parentTree,
+    parentNode,
     connectDragSource,
     //connectDragPreview, isDragging
   } = args
   return connectDragSource(
     <div
       style={{
         ...styles.dFlex,
-        ...(index === tree.children.length - 1 &&
+        ...(index === node.children.length - 1 &&
           !root.adding && { background }),
       }}
     >
-      {!(isRoot && tree.children.length === 1) && (
+      {!(isRoot && node.children.length === 1) && (
         <Operator
-          {...{ tree, child, root, parentTree, index, parent: state }}
+          {...{ node, child, root, parentNode, index, parent: state }}
```

Comment:
Given that we have a lot of instances of `node` and `child`, maybe we should go with `parent` instead of `parentNode`? Seems like we should either do that or rename `child` to `childNode`, but I think `parent` and `child` without adding node on the end are probably clear enough names